### ISSUE TITLE
Ui Feels slow / not so responsive

### ISSUE THREAD

**Issue by void95** [lllyasviel/stable-diffusion-webui-forge#1088]
When im Using the Gradio UI it feels slow, especially when i move the text cursor. Just kinda delayed. Someone has the same problem?

**Comment by tazztone** [comment]
i found i have high CPU usage whenever i have the tab open. even when forge is completely closed (!). is this some gradio issue?
![image](https://github.com/user-attachments/assets/4461f556-08de-4cf2-b5f2-82c3266177da)

**Comment by Bullseye110** [comment]
> When im Using the Gradio UI it feels slow, especially when i move the text cursor. Just kinda delayed. Someone has the same problem?

Yep, I have the same problem, but no one believed me anyway. The UI feels like you are in a video game with 2 FPS and 300 ping (so laggy).

**Comment by void95** [comment]
I mean its not a big deal but stills feels a little bit disturbing. Dont know why that happens.

**Comment by DeryabinIvan** [comment]
It's because of extensions. I heard they don't support Gradio 4 (current frontend), so disabling them fixes lagging issue.

**Comment by tazztone** [comment]
removed extensions and no more cpu spike. i had those installed:

facechain
IF_prompt_MKR
OneButtonPrompt
sd-civitai-browser-plus
sd-dynamic-prompts
sd-webui-infinite-image-browsing
sd-webui-prompt-all-in-one
sd-webui-reactor
ultimate-upscale-for-automatic1111

**Comment by Bullseye110** [comment]
> It's because of extensions. I heard they don't support Gradio 4 (current frontend), so disabling them fixes lagging issue.

Did that and it still feels laggy, but it's a little bit better without extensions.
